sqlx-core/src/pool/mod.rs

```diff
@@ -506,7 +506,7 @@ impl<DB: Database> Pool<DB> {
     /// As of 0.6.0, this has been fixed to use a separate atomic counter and so should be fine to
     /// call even at high load.
     ///
-    /// This previously called [`crossbeam::queue::ArrayQueue::len()`] which waits for the head and
+    /// This previously called `crossbeam::queue::ArrayQueue::len()` which waits for the head and
     /// tail pointers to be in a consistent state, which may never happen at high levels of churn.
```

Comment:
These four lines can be deleted since they're an implementation detail anyway.

Reply:
removed those lines